src/translations/en.json

```
@@ -107,6 +123,16 @@
         "spectators":"Spectators",
         "noSpectators":"Currently no spectators"
       }
+    },
+    "counter": {
+      "points": "POINTS",
+      "goal": "GOAL",
+      "kings": "0 Kings | 1 King | {count} Kings",
+      "yourTurn": "YOUR TURN",
+      "opponentTurn": "OPPONENT'S TURN"
+    },
```

As I mentioned above, let's reorganize these slightly
```suggestion
    "score": {
      "points": "POINTS",
      "goal": "GOAL",
      "kings": "0 Kings | 1 King | {count} Kings",
    },
    "turn": {
      "yourTurn": "YOUR TURN",
      "opponentTurn": "OPPONENT'S TURN"
    },
```